angular/projects/public-nrpti/src/app/records/records-list/records-list.component.html

```diff
@@ -1,39 +1,112 @@
-  <div *ngIf="!loading && tableData.totalListItems <= 0">
-    No Records found.
+  <div class="spinner-container" *ngIf="loading">
+    <div class="spinner-new rotating"></div>
   </div>
 
-  <div class="row mb-3">
-    <div class="col text-right">
-      <div>
-        <span class="mr-2 align-middle">Sort List By:</span>
-        <div ngbDropdown class="d-inline-block">
-          <button class="btn btn-primary" ngbDropdownToggle>
-            {{sortingDisplay[tableData.sortBy]}}
-          </button>
-          <div ngbDropdownMenu>
-            <button ngbDropdownItem (click)="changeSort('-dateIssued')">{{sortingDisplay['-dateIssued']}}
-              <i *ngIf="tableData.sortBy === '-dateIssued'" class="align-middle material-icons">
-                check_circle
-              </i></button>
-            <button ngbDropdownItem (click)="changeSort('+recordType')">{{sortingDisplay['+recordType']}}
-              <i *ngIf="tableData.sortBy === '+recordType'" class="align-middle material-icons">
-                check_circle
-              </i></button>
-            <button ngbDropdownItem (click)="changeSort('+issuedTo.fullName')">I{{sortingDisplay['+issuedTo.fullName']}}
-              <i *ngIf="tableData.sortBy === '+issuedTo.fullName'" class="align-middle material-icons">
-                check_circle
-              </i></button>
-            <button ngbDropdownItem (click)="changeSort('+location')">{{sortingDisplay['+location']}}
-              <i *ngIf="tableData.sortBy === '+location'" class="align-middle material-icons">
-                check_circle
-              </i></button>
+  <div *ngIf="!loading">
+    <section class="content-section">
+      <div class="content-grid" [ngClass]="{ 'show-advanced-filters': showAdvancedFilters }">
+        <div class="left-column">
+          <div class="search-filter__header">
+            <div class="inline-header">
+              <h5>Keyword Search</h5>
+              <i
+                class="material-icons grey-subtext"
+                matTooltip="summary, description, location, and issued to name fields"
+                aria-label="Icon that displays a tooltip when focused or hovered over"
+              >
+                help_outline
+              </i>
+            </div>
+            <div class="keyword-search">
+              <input
+                (keyup.enter)="keywordSearch()"
+                placeholder="Type keyword search words..."
+                [(ngModel)]="keywordSearchWords"
+              />
+              <button (click)="keywordSearch()">
+                <i class="material-icons">search</i>
+              </button>
+            </div>
+            <div>
+              <button [hidden]="showAdvancedFilters" class="btn btn-primary" (click)="openAdvancedFilters()">
+                Open Advanced Filters
+              </button>
+            </div>
           </div>
+          <div class="search-filter__content" [hidden]="!showAdvancedFilters">
```

Comment:
This and a number of other classes referenced in this file are not declared - we should be using bootstrap classes first, then looking for customizations.

Reply:
All of the classes should be declared.  

Some of them are cascading via scss, so a straight search for the entire class won't necessarily find anything. 

(In this case searching for 'search-filter' or  '__content' would yield search results.)

Reply:
Also, I went through and replaced a bunch of the scss I had added with bootstrap classes.

I'm not super familiar with all of the offerings of bootstrap, but I tried to cover the obvious ones like padding/margin/alignment/etc

Reply:
Perfect thanks - that's the preference!